FHEM/00_SIGNALduino.pm

```diff
@@ -1006,8 +1006,14 @@ sub SIGNALduino_parseResponse($$$) {
 sub SIGNALduino_ResetDevice($) {
 	my $hash = shift;
 	my $name = $hash->{NAME};
-	if (!defined($hash->{helper}{resetInProgress}))
-	{
+
+	if (IsDummy($name)) { # for dummy device
+		$hash->{DevState} = "initialized";
+		readingsSingleUpdate($hash, "state", "opened", 1);
+		return;
```

Comment:
Bei dem Return Wert würde ich eher undef übergeben wenn alles in Ordnung war.

Der Codeabschnitt könnte auch besser unterhalb der Logmeldung passen, was denkst Du?

Reply:
Das undef bei dem Return ändere ich ab. Kein Probelem.

>Der Codeabschnitt könnte auch besser unterhalb der Logmeldung passen, was denkst Du?

Ich sehe in der Sub nur 2 Logmeldungen welche innherhalb der Abfrage 
```
if (!defined($hash->{helper}{resetInProgress}))
	{
```
ist.

Können wir machen, somit erhalten wir noch zusätzlich die Ausgabe

```
2020.01.02 11:10:49 2: sduino_dummy: CloseDevice, closed
2020.01.02 11:11:00 3: sduino_dummy: ResetDevice, radinoCC1101
```
vom Dummy. Vorschlag ist gut. Ändere ich ab.